content/api/hub/architecture_pub_sub.md

```diff
@@ -0,0 +1,26 @@
+title = "Polyglot Publish-Subscribe"
+template = "render_hub_content_body"
+date = "2024-04-03T13:50:00Z"
+content-type = "text/plain"
+tags = ["publish-subscribe", "architecture", "polyglot"]
+
+[extra]
+author = "Thorsten Hans"
+type = "hub_document"
+category = "Architecture"
+language = "Polyglot"
```

Comment:
Noticed that the `Polyglot` language did not surface in the UI.
![Screenshot 2024-04-05 at 11 32 34](https://github.com/fermyon/developer/assets/9831342/d8b37b6e-3af4-41b0-ab33-fb2b6c1177e9)

Reply:
I added Polyglot to the `store.js` file for your convenience.
It now shows.

![Screenshot 2024-04-05 at 11 36 44](https://github.com/fermyon/developer/assets/9831342/e7a45efa-b16d-4d9f-8c3a-65cd6384522b)